Dockerfile

```diff
@@ -1,8 +0,0 @@
-FROM node:12.18.3-alpine
```

Comment:
Note that both "security hotspots" [identified](https://sonarcloud.io/summary/new_code?id=ginger-io_jay-z) by SonarCloud pertain to this file:

<img width="833" alt="image" src="https://user-images.githubusercontent.com/697964/236311374-d532cb11-e641-4149-bdd2-965c5f7b2b3b.png">

<img width="832" alt="image" src="https://user-images.githubusercontent.com/697964/236311455-923e51e9-faf9-4045-abd0-c10432630d06.png">